guides/beginners-course.rst

```diff
@@ -1,15 +1,255 @@
+
+Před začátkem
+-------------
+
+Než může kurz začít, potřebuješ místo, čas, mentory, účastníky, a obsah.
+
+Čas
+...
+
+Jak často chceš učit?
+
+    Nám se vyplatilo se sejít jednou za týden.
+    Intenzivní kurzy mohou být i častěji, ale je složitější sehnat dobrovolníky.
+    Méně časté lekce nedoporučujeme, pokud neexistuje doplňující program
+    (např. úkoly online).
+
+Kdy to bude?
+
+    Nám se osvědčily kurzy večerní, ve všední dny od 17:00 nebo 18:00.
+    Chceš-li nalákat pracující nebo studující, je to asi jediná reálná možnost.
+
+Jak dlouho?
+
+    Každá lekce našeho kurzu trvá dvě hodiny.
+    Po dvou-třech hodinách začíná být člověk unavený a učení už nejde tak dobře.
+
+
+Účastníci
+.........
+
+Kolik lidí chceš kurzem provést?
+
+    Naše začátečnické kurzy jsou pro zhruba 25 lidí.
+    Počítej s tím, že pokud je kurz zadarmo, budou lidi postupně odpadávat
+    – např. se jim změní pracovní situace, nebo jednoduše zjistí, že
+    programování není pro ně.
+
+    Klidně se může stát, že z 25 lidí celým kurzem projdou 3.
+
+Jaká je cílová skupina?
+
+    Není začátečník jako začátečník.
+    Chceš (jako my) učit lidi, kteří vůbec neumí progrmovat,
+    nebo ty kteří přicházejí z jiného jazyka?
+    Chceš učit vědce? Učitele? Testery? Středoškoláky?
+    Každé omezení cílové skupiny ti umožní kurz lépe přizpůsobit.
+
+    Je dobré když má každý účastník svůj laptop, který pak používá
+    i mimo lekce.
+
+
+Místo
+.....
+
+
+Podle toho, kolik plánuješ účastníků, vyber vhodné prostory.
+
+Sponzorované prostory
+
+    Pro komunitní kurz je dobré se zeptat IT firem, jestli by neposkytly na
+    večer zasedačku. Místo mají občas i školy či studentské unie.
+
+Placené prostory
+
+    Placené kurzy (nebo lépe, kurzy s nenulovým rozpočtem) se dají pořádat
+    v coworkingovém centru nebo školícím středisku.
+    Možností je spousta,
+
+
+Mentoři
+.......
+
+Hlavní kouč
+
+    Na každé lekci by měl být jeden „hlavní kouč”, který lekci řídí
+    a zná příslušné materiály.
+    Je dobré, když tuhle funkci po celé trvání kurzu zastává ten stejný
+    člověk, který má pak přehled o tom, co se probralo.
+    Pokud se hlavní koučové střídají, měl by každou lekci vést někdo, kdo
+    byl minimálně na lekci předchozí.
+
+Vedlejší koučové
+
+    Ostatní mentoři pomáhají (např. rozdávají papírové materiály),
+    a hlavně řeší konkrétní problémy.
+    V téhle roli nemusí být všichni experti. Naopak, pro účastníka
+    předchozího „běhu” začátečnického kurzu je koučování skvělá příležitost
+    jak si všechno zopakovat, a zjistit kolik se toho naučil :)
+    
+    Doporučujeme zhruba jednoho vedlejšího kouče na 3-5 účastníků.
+    Ze začátku, na instalaci a nastavení prostředí, je lepší jich pozvat spíš
+    víc; potom nebývá problém když občas někdo odpadne.
+
+    Koučové se dají nabrat na srazech, případně se dají oslovit programátoři
+    v okolních firmách.
+    Určitě koučování nabídni i účastníkům předchozích kurzů.
```

Comment:
Tohle by bylo dobré pak rozšířit, že je fakt hodně dobré, když se verbují původní účastníci.